src/frob.cc

```diff
@@ -63,6 +63,16 @@ unsigned long StringData(unsigned long addr) {
   // this works only if the filename is all ascii *fingers crossed*
   return addr + sizeof(PyASCIIObject);
 }
+#elif PYFLAME_PY_VERSION == 36
+namespace py36 {
```

Comment:
Does this actually need to be a new namespace? I think you could keep the `PYFLAME_PY_VERSION` guard, but reuse the existing `py3` namespace. Then you don't have to change `src/pyfrob.cc`.